pkgs/development/python-modules/sudachipy/default.nix

```
@@ -18,6 +18,8 @@ buildPythonPackage rec {
   pname = "sudachipy";
   inherit (sudachi-rs) src version;
 
+  patches = sudachi-rs.cargoPatches;
+
   cargoDeps = rustPlatform.fetchCargoVendor {
     inherit src;
     name = "${pname}-${version}";
```

I think patch needs to be applied to cargoDeps as well.

since cargoDeps sees old version then sudachipy requires newer (patched) version.

so it does not build.

```suggestion
  cargoDeps = rustPlatform.fetchCargoVendor {
    inherit src;
    name = "${pname}-${version}";
    patches = sudachi-rs.cargoPatches;
    hash = "sha256-NEWHASH";
```